### DISCUSSION TITLE
copying last section for left side/right side

### DISCUSSION THREAD

**Discussion by pixelbackwards1** [JordanSantiagoYT/FNF-JS-Engine#308] [Ideas]
i've always wanted to have this in js engine until months later, here's a reason why

![Screenshot 2024-05-04 140027](https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/166980779/980df10e-3ea9-49c2-b0f3-6d99edced39f)

charting a specific song that has for ex. spam charting on the left side and normal charting on the right side

sorry for this being too short and low effort!!

**Comment by JordanSantiagoYT** [comment]
there's already enough buttons in the section part, why add another 2

and uhh there's nobody whos done a chart editor tab with a subsection/extra page
